compass/landice/tests/antarctica/mesh.py

```diff
@@ -123,34 +123,46 @@ def run(self):
         dsMesh = convert(dsMesh, logger=logger)
         write_netcdf(dsMesh, 'antarctica_dehorned.nc')
 
+        mesh_filename = self.mesh_filename
         logger.info('calling create_landice_grid_from_generic_MPAS_grid.py')
         args = ['create_landice_grid_from_generic_MPAS_grid.py', '-i',
                 'antarctica_dehorned.nc', '-o',
-                'Antarctica.nc', '-l', levels, '-v', 'glimmer',
+                mesh_filename, '-l', levels, '-v', 'glimmer',
                 '--beta', '--thermal', '--obs', '--diri']
 
         check_call(args, logger=logger)
 
         logger.info('calling interpolate_to_mpasli_grid.py')
         args = ['interpolate_to_mpasli_grid.py', '-s',
                 'antarctica_8km_2020_10_20.nc',
-                '-d', 'Antarctica.nc', '-m', 'b']
+                '-d', mesh_filename, '-m', 'b']
         check_call(args, logger=logger)
 
         logger.info('Marking domain boundaries dirichlet')
         args = ['mark_domain_boundaries_dirichlet.py',
-                '-f', 'Antarctica.nc']
+                '-f', mesh_filename]
         check_call(args, logger=logger)
 
         logger.info('calling set_lat_lon_fields_in_planar_grid.py')
         args = ['set_lat_lon_fields_in_planar_grid.py', '-f',
-                'Antarctica.nc', '-p', 'ais-bedmap2']
+                mesh_filename, '-p', 'ais-bedmap2']
         check_call(args, logger=logger)
 
         logger.info('creating graph.info')
-        make_graph_file(mesh_filename='Antarctica.nc',
+        make_graph_file(mesh_filename=mesh_filename,
                         graph_filename='graph.info')
 
+        # create a region mask
+        mask_filename = 'imbie_regionMasks.nc'
```

Comment:
How about making this something like `f'{mesh_filename}_imbie_regionMasks.nc'?  (And similar for ismip6 mask filename)